### ISSUE TITLE
[Question] what exactly does "is_umbalance" in binary tasks?

### ISSUE THREAD

**Issue by gugatr0n1c** [Microsoft/LightGBM#225]
Hi,

there is several way how to deal with umbalance tasks. What exactly does this parameter here?

Some sampling?
Different objetictive/metric? Precision, Recall, Fscore? (Seems to be always logloss).
Something different?

In docs, there is only:
**is_unbalance, default=false, type=bool
used in binary classification. Set this to true if training data are unbalance.**

Can you describe it more in docs as well?
Thx

**Comment by wxchan** [comment]
if is_unbalance = true, weight of each positive sample / weight of each negative sample =  num_negative_sample / num_positive_sample

**Comment by gugatr0n1c** [comment]
thx
